cloud/amazon/ec2_eip.py

```diff
@@ -58,6 +58,12 @@
     required: false
     default: false
     version_added: "2.0"
+  private_ip_address:
```

Comment:
@rickmendes i can, but i'm following boto3 naming conventions here.  I'd prefer to mimic boto3 if possible.